### PR TITLE
Add ssl and authentication options

### PR DESCRIPTION
**Pull request by jgillis01**

### CODE REVIEW

**Comment by jgillis01** [comment]
Seems like it is failing with jruby-head:
There was a Java::JavaLang::NullPointerException while loading jackalope.gemspec

All others are passing, even jruby-19mode

**Comment by jszmajda** [comment]
Head builds can fail sometimes when head is broken.. Maybe try it again soon. Safe to merge otherwise IMO

File: README.md [hunk 1]
```diff
@@ -29,6 +29,22 @@ for rails app:
 
     jackalope -q my.queue config.ru
 
+all parameters:
+
+| Parameters    |                     Description                    | Default Value |
+|---------------|:--------------------------------------------------:|--------------:|
+| -q, --queue   | rabbitmq queue name for application communication  | default.queue |
+| -r, --host    |         rabbitmq server host or IP address         |     localhost |
```
Comment (jszmajda) [inline README.md]: You changed this to `-h` in the code

Comment (jgillis01) [inline README.md]: Thanks for catching that!  I will change this and commit

**Pull request merged**